### PR TITLE
Add missing admin role to env example on readme

### PR DESCRIPTION
**Pull request by RoyTakanen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,10 @@ AWARD_CHANNEL_ID=
 NO_REPORTS_ROLE_ID=
 AWARD_ROLE_ID=
 SILENCED_ROLE_ID=
+GIVEAWAY_DEFAULT_DURATION=
+GIVEAWAY_DEFAULT_WINNERS=
+GIVEAWAY_DEFAULT_PRIZE=
+GIVEAWAY_REACTION_EMOJI
```
Comment (Esinko) [inline README.md:46]: Shouldn't this line end with =?

Comment (RoyTakanen) [reply README.md:46]: Yeah, I did not focus enough to the copy pasting job :D

**Comment by RoyTakanen** [comment]
I also added database migrations commit because it was only one line edit.

**Comment by lajp** [comment]
LGTM, merging now.

**Pull request merged**